danceschool/vouchers/views.py

```diff
@@ -47,9 +47,9 @@ def dispatch(self,request,*args,**kwargs):
         if i.unpaid or i.amountPaid != self.amount:
             return HttpResponseBadRequest(_('Passed invoice is not paid.'))
 
-        return super(GiftCertificateCustomizeView,self).dispatch(request,*args,**kwargs)
+        return super(GiftCertificateCustomizeView, self).dispatch(request, *args, **kwargs)
```

Comment:
Does the project support Python 2?

Reply:
I believe we were trying to go the direction of Python3 but after looking through `requirements.txt` and the `README.md` I don't explicitly see a requirement for Python.

@merwok to clarify are you asking because it is being used in a way that is only supported by Python 3?

@leetucker do you mind chiming in?

Reply:
I’m asking because with Python 3, you can write `super().dispatch(request, etc)` which is quite nice.